### ISSUE TITLE
Option to open bookmark without anonymous mode

### ISSUE THREAD

**Issue by istudyatuni** [KotatsuApp/Kotatsu#432]
### Describe your suggested feature

It would be cool to have an option in bookmark's context menu to open bookmark without anonymous mode

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by istudyatuni** [comment]
Yes, I can, but it seems a little excessive (find that chapter and page) for such a simple action. I could try to do it myself, but I need your help to find where to do this, because this project is not so simple to understand

**Comment by Koitharu** [comment]
Ok, UI ideas are welcome)
